server/models.py

```diff
@@ -204,10 +206,19 @@ class Batch(BaseModel):
     storage_location = Column(String(200))
     tabulator = Column(String(200))
 
+    # For ballot polling audits, a batch is associated with a group of ballots
+    # sampled from this batch
     ballots = relationship(
         "SampledBallot", back_populates="batch", uselist=True, passive_deletes=True
     )
 
+    # For batch comparison audits, we sample the batch itself, and assign it to
+    # an audit board to be audited.
+    audit_board_id = Column(
+        String(200), ForeignKey("audit_board.id", ondelete="cascade"),
```

Comment:
this is where I would love knowing if it's nullable without having to remember the default.

Reply:
I can live with this :)